configure.ac

```diff
@@ -22,7 +22,7 @@ AC_SUBST(ODP_VERSION_API_MINOR)
 ##########################################################################
 m4_define([odph_version_generation], [1])
 m4_define([odph_version_major], [0])
-m4_define([odph_version_minor], [6])
+m4_define([odph_version_minor], [7])
```

Comment:
Since changes are not backwards compatible, the new version number should be 1.1.0. Minor number increment indicate backwards compatible changes.

Reply:
Fixed in v7.